ci/run-local.sh

```diff
@@ -16,6 +16,12 @@ case $(uname -o) in
     ;;
 esac
 
+here="$(dirname "$0")"
+cargo="$(readlink -f "${here}/../cargo")"
+"$cargo" audit --version || "$cargo" install cargo-audit --version 0.14.1
+grcov --version || "$cargo" install grcov --version 0.8.0
```

Comment:
How about we move these into `ci/do-audit.sh` and `scripts/coverage.sh` respectively?

Maybe put the versions in `ci/rust-version.sh` while we're at it?